packages/nimble-components/src/wafer-map/specs/features/padding.md

```diff
@@ -0,0 +1,38 @@
+# Nimble Wafer Map Die Padding
+
+## Problem Statement
+
+The Wafer Map displays dies on a canvas inside a circular border with a dynamic padding size based on the quantity of dies.
+The padding size needs to be regulated and specific, therefore a public input parameter will be used to allow the setting of padding size.
+
+## Links To Relevant Work Items and Reference Material
+
+[Wafer Map Spec](../README.md)
+
+[D3 ScaleBand](https://observablehq.com/@d3/d3-scaleband)
+
+[D3 ScaleQuantile](https://observablehq.com/@d3/quantile-quantize-and-threshold-scales)
+
+## Implementation / Design
+
+![Die Padding](resources/padding.png)
+
+The padding will influence the size of the borders between dies and will be represented as a percentage of the die size.
```

Comment:
So if I understand correctly the size of the die are not changing. just the spacing between them?

If that's correct is this a padding per die that is being represented or a gap between dies?

A padding per die would mean that each die would get some padding around them, so with a diePadding of 1 the actual space between the die will be 2*dieSize

A gap would mean with a diaPadding of 1 the actual space between die will be the full size of a die.

![image](https://user-images.githubusercontent.com/1588923/220165553-fa4e24a5-f585-4267-a7dd-fa553e86aa23.png)

If this represents a gap then I think we should go for the name `dieGap` instead. If this actually represents per die padding then the current name makes sense.